lib_web/context.ml

```diff
@@ -2,6 +2,14 @@ open Lwt.Infix
 
 let cookie_key = "__session"
 
+let img_ocaml_logo = "/images/ocaml%20logo.png"
```

Comment:
Why should OCurrent pipelines have an OCaml logo? Performing CI for OCaml projects isn't the only use for pipelines.

Reply:
I have renamed this to dashboard-logo.png.  Pipelines have full control over this image (because they control the docroot that images are served from) so with a generic name I think that all use cases are covered.